src/features/currentUser/currentUser.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+    CurrentUser: {}
+}
+
+export const CurrentUserSlice = createSlice({
+    name: 'currentUser',
+    initialState,
+    reducers: {
+        changeCurrentUser: (state, action) => {
```

Comment:
instead of copying user I suggest to only save current user index. so you have an array of users `allAccountsList`. when you want to set current user you just save the index of the current user and write a custom selector that will return smth like
```ts
const currentUserSelector = (state) => state.allAccountsList[state.currentUserIndex]
```
this keeps your store in a more normalized state preventing unneccessary duplicates